app/src/main/java/ca/chronofit/chrono/stopwatch/StopwatchFrag.kt

```diff
@@ -154,18 +152,21 @@ class StopwatchFrag : Fragment() {
     }
 
     private fun lap() {
-        lapCount += 1
+        if (lapCount == 0) {
+            laps.add(LapObject())
```

Comment:
Can this be moved into the onCreate() function so that it is only ever called once?

Reply:
Yep

Reply:
Done